### DISCUSSION TITLE
Unable to install and use older Hive-JDBC driver

### DISCUSSION THREAD

**Discussion by ruffyleaf** [dbeaver/dbeaver#20149] [General]
#### System information: 
- Operating system (distribution) and version
-- osx Big Sur, 11.3
- DBeaver version
-- Version 21.0.4.202105021514, Community
- Java version 
-- Java(TM) SE Runtime Environment (build 1.8.0_212-b10)
- Additional extensions

#### Connection specification:
- Database name and version
-- Hive 1.1.0 on VirtualBox Cloudera appliance
- Driver name
-- Hive-JDBC driver
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 
-- No. Tried using SSH tunnel, but still didn't work.

#### Describe the problem you're observing:
Unable to connect to Hive using an older Hive-JDBC driver. Tried to download and install the older versions but still unable to connect to hive. It doesn't seem to select the driver in the jar.

#### Steps to reproduce, if exist:
1. Add New Hive connection
2. Add hive-jdbc 1.1.0 .jar file into the driver list
3. Test connection fails

#### Include any warning/errors/backtraces from the logs

2040 !MESSAGE No suitable driver found for jdbc:hive2://localhost:10000/retail_db
2041 !SUBENTRY 1 org.jkiss.dbeaver.model 4 0 2021-05-20 09:38:54.651
2042 !MESSAGE No suitable driver found for jdbc:hive2://localhost:10000/retail_db
2043 !STACK 0
2044 java.sql.SQLException: No suitable driver found for jdbc:hive2://localhost:10000/retail_db
2045         at java.sql/java.sql.DriverManager.getConnection(Unknown Source)
2046         at java.sql/java.sql.DriverManager.getConnection(Unknown Source)
2047         at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.lambda$0(JDBCDataSource.java:174)
2048         at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:195)
2049         at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.openConnection(GenericDataSource.java:135)
2050         at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:101)
2051         at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:100)
2052         at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:59)
2053         at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:109)
2054         at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.<init>(GenericDataSource.java:107)
2055         at org.jkiss.dbeaver.ext.hive.model.HiveDataSource.<init>(HiveDataSource.java:41)
2056         at org.jkiss.dbeaver.ext.hive.model.HiveMetaModel.createDataSourceImpl(HiveMetaModel.java:48)
2057         at org.jkiss.dbeaver.ext.generic.GenericDataSourceProvider.openDataSource(GenericDataSourceProvider.java:106)
2058         at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:897)
2059         at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:70)
2060         at org.jkiss.dbeaver.runtime.jobs.ConnectionTestJob.run(ConnectionTestJob.java:103)
2061         at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
2062         at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

**Comment by kseniiaguzeeva** [comment]
Thank you for the report.

**Comment by Matvey16** [comment]
@ruffyleaf Hello. Could you more specifically describe how did you add your driver? I was not able to reproduce it and Dbeaver tried to use the 1.1.0 driver. But this driver is pretty old and it won't work in recent DBeaver version.
